src/cloud/project/new-relic.md

```diff
@@ -88,8 +89,21 @@ To find your New Relic license key:
 
 1. Copy the license key.
 
+### Second Option
+
+1. Open your account on New Relic Page [account page](https://login.newrelic.com/login).
```

Comment:
As a practice, we try not to duplicate instructions for using third-party applications in the Magento documentation. The procedure in the New Relic documentation is thorough and complete.  Additionally, if New Relic makes UI or workflow changes, their documentation will be the most current and up-to-date.

I included the link to the New Relic instructions for copying the license key above: https://docs.newrelic.com/docs/accounts/install-new-relic/account-setup/license-key#finding